README.md

```diff
@@ -46,12 +46,274 @@ setting up a database inside an enclave is very well possible. This sample shows
 table, insert records into it, and select records from the table. This also shows how persisted records can be retrieved by
 the enclave once the host is re-started.
 
-### Initial SetUp
+# Write your first Conclave application in 5 mins
 
-In order to compile, and run these samples successfully, kindly download the [conclave SDK](https://www.conclave.net/get-conclave/) and update 
-the conclaveRepo property in gradle.properties to point to this path.
+This tutorial can be used to get started with Conclave in 5 mins. We will create a sample in which the client can send
+a string to the enclave, the enclave will reverse it and send it back to the client.
 
-    conclaveRepo=add_path_to_repo
+#Step 1: Prerequisites
+You need [JDK](https://www.openlogic.com/openjdk-downloads) 8 or 11 installed in order to build and run the app.<br />
+Grab a copy of Conclave-SDK from [conclave.net](https://www.conclave.net/get-conclave/)
+We will use the conclave init tool for bootstrapping Conclave projects, reducing the amount of boilerplate you need to write.
+</br>You will find conclave-init.jar in the tools directory of the Conclave SDK.
+
+# Step 2: Create the project template using the conclave-init tool
+
+To create a new Conclave project, run the following command:
+
+    java -jar /path/to/conclave-sdk/tools/conclave-init.jar 
+    --package com.megacorp 
+    --enclave-class-name ReverseEnclave 
+    --target ./reverse-conclave-app
+
+A new project would have been created in the reverse-conclave-app directory, as specified by --target ./reverse-conclave-app.
```

Comment:
does it create the directory if it doesn't already exist?

Reply:
yes, it creates the directory if it doesn't exist.